docs/remote/tunnels.md

```diff
@@ -141,6 +141,14 @@ For instance, right now you can have 5 tunnels registered for your account. If y
 
 If you are hitting a limit and would like to discuss getting more usage, we'd love to learn more. Please reach out to `vscodeserver@microsoft.com` with information on your scenario and how much usage you need.
 
+### How can I ensure I keep my tunnel running?
+
+You have a few options:
+* Use the `service` command, i.e. `code tunnel --service`, to run as a service
```

Comment:
This flag doesn't exist -- the user would run `code tunnel service install` and `code tunnel service uninstall` to install and remove them

Reply:
Ha whoops, thanks! Updated.